src/components/pages/Dashboard/Dashboard.js

```diff
@@ -60,39 +61,38 @@ const columns = [
   },
 ];
 
-const data = [
-  {
-    key: '1',
-    ticketID: '1234',
-    ticketType: 'Application',
-    dateSubmitted: '1/21/2022',
-  },
-  {
-    key: '2',
-    ticketID: '1233',
-    ticketType: 'Resource Request',
-    dateSubmitted: '1/20/2022',
-  },
-  {
-    key: '3',
-    ticketID: '1223',
-    ticketType: 'Escalation',
-    dateSubmitted: '1/19/2022',
-  },
-  {
-    key: '4',
-    ticketID: '1200',
-    ticketType: 'Application',
-    dateSubmitted: '1/10/2022',
-  },
-];
-
 function onChange(pagination, filters, sorter, extra) {
   console.log('params', pagination, filters, sorter, extra);
 }
 
 // TODO: make Ant Design Statistics pull ticket totals from ticket tables
 const Dashboard = props => {
+  const [tickets, setTickets] = useState([]);
+
+  useEffect(() => {
+    const getTickets = () => {
+      axiosWithAuth()
+        .get('/resource-tickets')
+        .then(res => {
```

Comment:
Great job implementing the axios call to get real data from the backend. Maybe consider adding a .catch for error handling

Reply:
I would also love to see a `.catch` for error handling here - let's make a note to add it when we circle back to this feature to iron out any kinks.